.github/workflows/windows.yml

```diff
@@ -0,0 +1,53 @@
+name: CI
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  windows:
+    name: Windows
+    runs-on: windows-latest
+    timeout-minutes: 30
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up JDK 11
+      uses: actions/setup-java@v1
+      with:
+        java-version: '11'
+
+    - name: Setup Node.js environment
+      uses: actions/setup-node@v2
+      with:
+        node-version: 12
+        
+    - name: Install Node.js modules
+      run: npm install
+
+    - name: Install Gulp
+      run: npm install -g gulp --force
+
+    - name: Install VSCE
+      run: npm install -g vsce
+
+    - name: Lint
```

Comment:
Just use one instance to run linting is enough. Check https://github.com/microsoft/vscode/blob/main/.github/workflows/ci.yml